modules/tinymce/src/themes/silver/main/ts/ReadOnly.ts

```diff
@@ -60,32 +49,37 @@ export const toggleToReadOnly = (uiComponents: RenderUiComponents, readonly: boo
   });
 };
 
-export const setupReadonlyModeSwitch = (editor: Editor, uiComponents: RenderUiComponents) => {
+const setupReadonlyModeSwitch = (editor: Editor, uiComponents: RenderUiComponents) => {
   editor.on('init', () => {
     // Force an update of the ui components disabled states if in readonly mode
-    if (editor.mode.isReadOnly()) {
+    if (editor.readonly) {
       toggleToReadOnly(uiComponents, true);
     }
   });
 
-  editor.on('SwitchMode', () => toggleToReadOnly(uiComponents, editor.mode.isReadOnly()));
+  editor.on('SwitchMode', () => toggleToReadOnly(uiComponents, editor.readonly));
```

Comment:
Another place that was changed to `editor.readonly` that possibly shouldn't have been changed?